guides/release/configuring-ember/debugging.md

```diff
@@ -1,6 +1,9 @@
 Ember provides several configuration options that can help you debug problems
 with your application.
 
+## Ember Inspector
```

Comment:
Can we update lines 1 and 2 to something like:

```
Ember provides a browser extension and several configuration options
to help you debug your application.
```

Initially, I was confused by why the issue asked to mention Ember Inspector in a chapter that discussed configuration options. By updating the introductory paragraph, I think it'll make more sense to mention Ember Inspector.

Reply:
Nice catch, I'll make that change